x-pack/plugins/cases/server/routes/api/utils.ts

```diff
@@ -32,3 +32,23 @@ export function wrapError(error: any): CustomHttpResponseOptions<ResponseError>
 }
 
 export const escapeHatch = schema.object({}, { unknowns: 'allow' });
+
+export const getCaseUrl = (url: string | string[] | undefined, caseId: string): string | null => {
+  const caseUrl = Array.isArray(url) ? url[0] : url;
+  if (caseUrl != null) {
+    // split at case id to get cases base url. if we split at ? we might get the comment or whatever else included in the url
+    let possibleCaseUrl;
+    let appName;
+    if (caseUrl.includes('app/security')) {
```

Comment:
@XavierM is there somewhere more official i should be getting the app url value to compare or is this chill?

Reply:
You could split with a regex here so we don't need to hard-code this:

```js
"app/security/foo/bar".split(/app\/\w+/)
(2) ["", "/foo/bar"]
"app/observability/foo/bar".split(/app\/\w+/)
(2) ["", "/foo/bar"]
```